src/install/config/cassandra.yaml

```diff
@@ -0,0 +1,81 @@
+agentName: cassandra
+agentType: integration
+title: 'Cassandra monitoring integration'
+metaDescription: 'Install the Cassandra integration'
+introFilePath: 'src/install/cassandra/intro.mdx'
+appInfo: 
+  - optionType: dropdown1
+    label: ''
+    placeholder: 'Select your environment'
+    options: 
+      - value: 'linux'
+        displayName: 'Linux'
+        logo: 'linux'
+      - value: 'ecs'
+        displayName: 'ECS'
+        logo: 'amazonECS'
+      - value: 'kubernetes'
+        displayName: 'Kubernetes'
+        logo: 'k8'
+  - optionType: deployment
+    label: "Choose your install path"
+    placeholder: "Choose one:"
+    options:     
+      - value: "apt"
```

Comment:
Since we recommend guided install in the intro, I'd cut out the `recommendedGuided: true` in these